app/features/modelling/dev-classifier/index.js

```diff
@@ -0,0 +1,43 @@
+class DevClassifierModel {
+  constructor(canvas, { frame }) {
+    this.ctx = canvas.getContext('2d');
+    this.frame = frame;
+  }
+  update(annotations, vBD) {
+    this.ctx.clearRect(0, 0, this.ctx.canvas.width, this.ctx.canvas.width);
+    for (let i = 0; i < annotations.length; i += 1) {
```

Comment:
Can `forEach` be used here instead of the for loops? 

Here's a good comparison on why you'd usually want to use `forEach` over a for loop: https://thejsguy.com/2016/07/30/javascript-for-loop-vs-array-foreach.html

Reply:
This file is no longer used, so I'll remove it. I used the for loop for performance, but annotations is so short it probably wouldn't matter.
Article was a good read, thanks!